### PR TITLE
leaderboard updates for mitch

### PR DESCRIPTION
**Pull request by owocki**
1. adds the ability to filter on org name
2. adds num bounties in addition to just the amount funded

screencapture-localhost-8000-leaderboard-quarterly_orgs-2018-05-25-14_00_18

### CODE REVIEW

**Comment by mkosowsk** [comment]
@owocki Thanks a lot! Please ping me here when this dude is merged, this will really help my workflow :)

**Comment by owocki** [comment]
@mkosowsk https://gitcoin.co/leaderboard/quarterly_orgs

File: app/marketing/models.py [hunk 1]
```diff
@@ -164,6 +164,8 @@ class LeaderboardRank(SuperModel):
     leaderboard = models.CharField(max_length=255)
     amount = models.FloatField()
     active = models.BooleanField()
+    count = models.IntegerField(default=0)
```
[the remark this answers is not in the document]
Comment (owocki) [reply app/marketing/models.py]: this is where it's computed -> https://github.com/gitcoinco/web/pull/1275/files#diff-c672b9ab5fac2576200d174f3f05baf0R80

we'd have to change the way we store / compute the info in this PR to do this.  not sure the performance implications, but right now the `assemble_leaderboards` script is fairly fast.  its about 15s for the script to run once a day

**Comment by mkosowsk** [comment]
@owocki Excellent, thanks a million!

**Pull request merged**